record/record.app.js

```diff
@@ -10,10 +11,15 @@
         'ui.bootstrap'
     ])
 
-    .run(['ERMrest', function (ERMrest) {
+    .run(['ERMrest', 'ConfigUtils', 'UriUtils', '$window', function (ERMrest, ConfigUtils, UriUtils, $window) {
         ERMrest.onload().then(function () {
-            angular.element(document).ready(function(){
-                angular.bootstrap(document.getElementById("record"), ["chaise.record"]);
+            var urlParts = UriUtils.extractParts($window.location);
```

Comment:
I'm a bit confused. If we're doing this logic here and not in the `config.js`, then what's the whole point of `config.js`?

By the way calling `catalogs.get` will introspect the whole catalog and since we just want the annotation, we might want to create another specific api that only calls `catalog/N` and returns the annotation. Although since we're always going to introspect the schemas that shouldn't matter.

Reply:
We have to pass all the `cid`, `pid`, and `wid` to the getServer function.